Userland/Libraries/LibCore/System.cpp

```diff
@@ -1205,6 +1198,13 @@ ErrorOr<Vector<gid_t>> getgroups()
     return groups;
 }
 
+ErrorOr<void> setgroups(Span<gid_t const> gids)
```

Comment:
Please don't move code, if you are going to move code put it in its own commit, not mixes with other functional changes.

Reply:
That includes moving it out of ifdefs 😁